static/src/javascripts/lib/config.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import pad from 'lib/pad';
+// eslint-disable-next-line guardian-frontend/global-config
+const config = window.guardian.config;
```

Comment:
feel like we should check the objects we're merging here will not overwrite each other... maybe that's out of the scope of this pr?